sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleFacadeEjb.java

```diff
@@ -1153,13 +1282,15 @@ public static SampleReferenceDto toReferenceDto(Sample entity) {
 	private void onSampleChanged(SampleDto existingSample, Sample newSample, boolean syncShares) {
 
 		// Change pathogenTestResultChangeDate if the pathogen test result has changed
-		if (existingSample != null
-			&& existingSample.getPathogenTestResult() != null
-			&& existingSample.getPathogenTestResult() != newSample.getPathogenTestResult()) {
+		if (existingSample == null
+				|| existingSample.getPathogenTestResult() != newSample.getPathogenTestResult()) {
+			
 			Date latestPathogenTestDate = pathogenTestFacade.getLatestPathogenTestDate(newSample.getUuid());
-			if (latestPathogenTestDate != null) {
-				newSample.setPathogenTestResultChangeDate(latestPathogenTestDate);
-			}
+			Date changeDate = latestPathogenTestDate != null
```

Comment:
Since you're already doing changes here - what if the pathogen test result of an existing sample is changed without a pathogen test being created? Shouldn't we use the current date to update the pathogen test result change date then, since the sample date time possibly won't be accurate anymore?